VultisigApp/VultisigApp/Chains/evm.swift

```diff
@@ -77,18 +77,19 @@ class EVMHelper {
         guard let intChainID = Int(coin.chainId) else {
             throw HelperError.runtimeError("fail to get chainID")
         }
-        guard case .Ethereum(let maxFeePerGasWei,
+        guard case .Ethereum(let baseFee,
                              let priorityFeeWei,
                              let nonce,
                              let gasLimit) = keysignPayload.chainSpecific
         else {
             throw HelperError.runtimeError("fail to get Ethereum chain specific")
         }
+        let maxFeePerGas = baseFee + priorityFeeWei
```

Comment:
I think this is the place that will break IOS~Android cross signing, correct? 
Why not calculate maxFeePerGas from outside , and then pass it in , like we did before?